catalog/core/catalog-core-metacardgroomerplugin/src/main/java/ddf/catalog/plugin/groomer/metacard/StandardMetacardGroomerPlugin.java

```diff
@@ -72,6 +73,15 @@ protected void applyCreatedOperationRules(CreateRequest createRequest, Metacard
             aMetacard.setAttribute(new AttributeImpl(Metacard.TAGS,
                     Collections.singletonList(Metacard.DEFAULT_TAG)));
         }
+
+        // upon creation of the metacard, generate attributes for created, modified dates
+        aMetacard.setAttribute(new AttributeImpl(ddf.catalog.data.types.Metacard.CREATED, now));
+        LOGGER.debug("Applying {} attribute with value {} to metacard [{}].",
+                Metacard.CREATED, now, aMetacard.getId());
+
+        aMetacard.setAttribute(new AttributeImpl(ddf.catalog.data.types.Metacard.MODIFIED, now));
+        LOGGER.debug("Applying {} attribute with value {} to metacard [{}].",
```

Comment:
Don't need to duplicate this message string. Could extract a constant or refactor the whole logger call into a utility method.